scripts/extract/parl_json.py

```diff
@@ -28,9 +37,25 @@ def get_json(url):
     return response
 
 
-def save_json(response_json, filepath):
-    with open(filepath, "w") as file:
-        json.dump(response_json, file)
-        print(f"JSON saved to: {filepath}\n")
+def upload_json(response_json, file, folder_id):
```

Comment:
nice work with this! can we put a comment here to link to the folder where files are uploaded?

Reply:
yes we can!